api/views/admin_auth.py

```diff
@@ -30,3 +51,40 @@ def register():
             "message": 'Bad request parameters: {}'.format(data['message'])
         }
         return make_response(jsonify(response_object), 400)
+
+
+@admin_auth.route('/admin/auth', methods=['POST'])
+def auth_user():
+    """ Endpoint to authorize users """
+    data = validate_user(request.get_json())
+    if data['ok']:
+        data = data['data']
+        user = users.find_one({
+            "email": data["email"]
+        })
+
+        if user and flask_bcrypt.check_password_hash(user['password'], data['password']):
+            del user['password']
+            access_token = create_access_token(identity=data)
+            refresh_token = create_refresh_token(identity=data)
+            user['token'] = access_token
+            user['refresh'] = refresh_token
+            
+            response_object = {"status": True, "data": user}
+            return make_response(jsonify(response_object), 200)
+        else:
+            response_object = {"status": False, "message": "Invalid username or password"}
+            return make_response(jsonify(response_object), 401)
+    else:
+        response_object = {"status": False, "message": 'Bad request parameters: {}'.format(data['message'])}
+        return make_response(jsonify(response_object), 400)
```

Comment:
The `else: "invalid username or password"` block only checks for bad password - can you make it so that exact block will execute in the case where an unknown username is passed as well?